concept-anchor.md

```diff
@@ -0,0 +1,51 @@
+## Title
+Explaining InnerSource to Management by anchoring it to Agile / DevOps / Lean
+
+## Patlet
+Those wanting to bring InnerSource to organizations have difficulty starting a discussion with management. This can sometimes be made easier by anchoring InnerSource to more well known concepts such as Agile / DevOps / Lean.
+
+## Problem
+Often InnerSource is a term that's novel to organizations, making it often difficult to start a conversation with management about adopting InnerSource.
+
+## Context
+An individual sees an opportunity for InnerSource to make an impact within an organization. They would like to discuss InnerSource with management. Being a relatively traditional and bureaucratic company, it is difficult to broach the subject of InnerSource with management as they are reluctant to engage on a topic that they've neither heard of nor know anything about.
+
+These organizations might have an existing strategy, such as Agile, Lean, or DevOps transformation. To gain mindshare, it might be useful to start with these existing, well known concepts and build InnerSource on top of these concepts.
+
+## Forces
+* Management are reluctant to talk to you unless you can tie projects in to an existing corporate strategy.
+* Starting to explain InnerSource may be difficult unless one starts with a point decision makers are familiar with.
+* Agile/DevOps/Lean have already been heavily marketed and are commonly discussed terms in corporations, conferences, magazines etc.
+
+## Solutions
+### Agile
+There are several approaches that can be taken in the context of Agile depending on who's listening. The following main concepts are from [Wikipedia](https://en.wikipedia.org/wiki/Agile_software_development)
+* *Incremental and evolutionary processes* - Submitting pull requests for early feedback, or creating an issue for early stakeholder discussion aids this process, allowing ideas to be shaped and defined in the very early conceptual stages, thus being able to nourish good ideas and filter out unfeasible ideas before there are any significant costs associated with them.
+* *Efficient and face-to-face communication* - This part will be difficult as communications are not face to face. However they are efficient and tend to be very inclusive. It's possible to cite that this is how much of the software that runs the internet was successfully coordinated and built so there's some virtue to doing it the IS way, which is efficient but online communication. Another point of argument is that there's no better way to communicate your expectations than to actually implement it, making the process extremely effective
```

Comment:
Suggestion: Given the "if it isn't written down and announced on the mailing list, it didn't happen" cultural norms of IS, we could mention that that the documentation heavy nature of IS processes mean that when key people depart an organization, they do not take the knowledge of the software with them.

Reply:
Good idea, do you have the appropriate wording in mind? If not I'll add something in a few hours, thank you for your valuable comments!

Reply:
Here is my proposal for this item:
* *Efficient and face-to-face communication* - Many enterprises still implement collaboration models centered around face to face collaboration with teams being colocated. It is possible that management may view the "If it isn't written down and announced on the mailing list, it didn't happen" cultural norm of IS as unnecessary. 

However, one can highlight benefits to corporations from implementing IS practices. 
a) Business continuity in managing "Key man risk" 
When key people depart, the knowledge does not go out of the door with them. Robust documentation including decision records/RFCs and the like capture the rationale for important project design decisions and are key to continued productivity of new employees within the scrum team. 

b) Fostering productivity
While the corporation's strategy may involve colocating employees who collaborate with each other regularly, it may still be a global corporation. Robust CI/CD and documentation facilitates collaboration and productivity across time zones and geographical locations. It eliminates barriers between teams and the production environment.